src/Build/Resources/Strings.resx

```diff
@@ -463,6 +463,10 @@
     <value>MSB4187: Failed to receive a response from the task thread in the timeout period "{0}" ms. Shutting down.</value>
     <comment>{StrBegin="MSB4187: "}</comment>
   </data>
+  <data name="UnhandledMSBuildError" xml:space="preserve">
+    <value>This is an unhandled exception in MSBuild -- PLEASE FILE AN ISSUE AT https://github.com/dotnet/msbuild/issues/new.
```

Comment:
Let's make an aka.ms link for this so we can easily retarget it if necessary

```suggestion
    <value>This is an unhandled exception in MSBuild -- PLEASE FILE AN ISSUE AT https://aka.ms/msbuild/unhandled.
```

I also think we might want to change the wording from "please file an issue" to "please search for and upvote an issue or if there isn't one file a new one" if we can figure out a way to say that that is less mean.

Reply:
So should that aka.ms/ link point [here](https://github.com/dotnet/msbuild/issues) instead?

I would definitely prefer they search for existing issues first, but telling them to search first might deter people from filing issues at all, which biases me a little towards the current wording. How about:
PLEASE UPVOTE AN EXISTING ISSUE OR FILE A NEW ONE AT...
?

Reply:
We should make a specific issue template for this error in this repo, and have the link point to issue create with that particular issue template preselected.  The SDK does this:

* [here's where the 'new issue' button takes you](https://github.com/dotnet/sdk/issues/new/choose)
* [specific classes of issues have their own templates](https://github.com/dotnet/sdk/issues/new?assignees=&labels=&template=20_feature_request.md)

We could then update the aka.ms link to point to the template for unhandled errors, with instructions on what to search for, data to provide on the issue, etc, etc.

Reply:
For whoever takes this on, some relevant PR's:

https://github.com/dotnet/msbuild/pull/5380
https://github.com/dotnet/msbuild/pull/6557
https://github.com/dotnet/msbuild/pull/5521

Reply:
I made a separate PR (#7392) for making the unhandled exception issue. That should mean we can merge the two independently and redirect the aka.ms link as necessary later.

By the way, what's the link for changing those? I wrote down that it's just https://aka.ms, but that took me to the benefits page.

Reply:
raw `aka.ms` takes you to the dashboard for updating them, right now Rainer, Marc, and I are set as the owners, so I think only we can see/change it

Reply:
I'm dumb. I typed aka.ms and hit enter without noticing that it autocompleted...
Apparently I most use the vanity links to log all my 2/4 hour absences 😁